gandalf.js

```diff
@@ -6,60 +6,54 @@ const firstFiveElements = array.slice(0, 5);
  *
```

Comment:
SCRIPT-I appreciate how well-structured and clean your code is. It's very easy to follow!